kessler/time_utils.py

```diff
@@ -0,0 +1,81 @@
+import datetime
+def getCcsdsTimeFormat(timeString):
+    '''
+    Adapted by Andrew Ng, 18/3/2022.
+    Original MATLAB source code found at: https://github.com/nasa/CARA_Analysis_Tools/blob/master/two-dimension_Pc/Main/TransformationCode/TimeTransformations/getCcsdsTimeFormat.m
+    The CCSDS time format is required to be of the general form
+    yyyy-[mm-dd|ddd]THH:MM:SS[.F*][Z]
+    (1) The date and time fields are separated by a "T".
+    (2) The date field has a four digit year followed by either a two digit 
+        month and two digit day, or a three digit day-of-year.  
+    (3) The year, month, day, and day-of-year fields are separated by a dash.
+    (4) The hours, minutes and seconds fields are each two digits separated 
+        by colons.
+    (5) The fraction of seconds is optional and can have any number of
+        digits.
+    (6) If a fraction of seconds is provided, it is separated from the two
+        digit seconds by a period.
+    (7) The time string can end with an optional "Z" time zone indicator
+    '''
+    timeFormat = []
+    numT = timeString.count('T')
+    if numT == -1:
+        # Case when this is 'T' does not exist in timeString
+        print(f"*** Error -- Invalid CCSDS time string: {timeString}")
+        print(f"    No 'T' separator found between date and time portions of the string\n")
+        return
+    elif numT > 1:
+        print(f"*** Error -- Invalid CCSDS time string: {timeString} \n")
+        print(f"    More than one 'T' separator found between date and time portions of the string\n")
+        return
+    idxT = timeString.find('T')
+    if idxT ==10:
+        timeFormat = "yyyy-mm-ddTHH:MM:SS"
+    elif idxT ==8:
+        timeFormat = "yyyy-DDDTHH:MM:SS"
+    else: 
+        print(f"*** Error -- Invalid CCSDS time string: {timeString} \n", timeString)
```

Comment:
Why not using only:
```
print(f"*** Error -- Invalid CCSDS time string: {timeString} \n")
```
?

Reply:
Also, if this should return an error, I think you should `raise` a `RuntimeError` here, by returning the error message above

Reply:
Good spot for the errors, I got mixed up between Python's f string and MATLAB's sprintf function!

Agreed with the error raising as well.